jsapp/xlform/src/view.icons.coffee

```diff
@@ -46,6 +46,8 @@ module.exports = do ->
   addIconToRow(constants.QUESTION_TYPES.hidden, "r6")
   addIconToRow(constants.QUESTION_TYPES.file, "r6")
   addIconToRow(constants.QUESTION_TYPES.range, "r6")
+  # row 7
+  addIconToRow(constants.QUESTION_TYPES.xml_external, "r7")
```

Comment:
I've done these changes but changed the constant and it's id to `xml-external`, it was crashing before because in the backend it's stored as `xml-external` so I don't know why we had two different names for it :shrug: